.github/ISSUE_TEMPLATE/NEW_RELEASE.md

```diff
@@ -18,31 +18,35 @@ Please do not remove items from the checklist
   - [ ] An OWNER pushes the new release branch with
         `git push release-$MAJ.$MIN`
 - [ ] Update things like README, deployment templates, docs, configuration, test/e2e flags.
-      Submit a PR against the release branch.
-- An OWNER prepares a draft release
-  - [ ] Create a draft release at [Github releases page](https://github.com/kubernetes-sigs/kueue/releases).
+      Submit a PR against the release branch: <!-- example #211 #214 -->
+- [ ] An OWNER [prepares a draft release](https://github.com/kubernetes-sigs/kueue/releases)
   - [ ] Write the change log into the draft release.
   - [ ] Run
       `make artifacts IMAGE_REGISTRY=k8s.gcr.io/kueue GIT_TAG=$VERSION`
       to generate the artifacts and upload the files in the `artifacts` folder
       to the draft release.
-- [ ] An OWNER runs
+- [ ] An OWNER creates a signed tag running
      `git tag -s $VERSION`
       and inserts the changelog into the tag description.
+      To perform this step, you need [a PGP key registered on github](https://docs.github.com/en/authentication/managing-commit-signature-verification/checking-for-existing-gpg-keys).
 - [ ] An OWNER pushes the tag with
       `git push $VERSION`
   - Triggers prow to build and publish a staging container image
       `gcr.io/k8s-staging-kueue/kueue:$VERSION`
-- [ ] Submit a PR against [k8s.io](https://github.com/kubernetes/k8s.io), updating `k8s.gcr.io/images/k8s-staging-kueue/images.yaml` to [promote the container images](https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io#image-promoter) to production.
+- [ ] Submit a PR against [k8s.io](https://github.com/kubernetes/k8s.io), 
+      updating `k8s.gcr.io/images/k8s-staging-kueue/images.yaml` to
+      [promote the container images](https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io#image-promoter)
+      to production: <!-- example kubernetes/k8s.io#3612-->
 - [ ] Wait for the PR to be merged and verify that the image `k8s.gcr.io/kueue/kueue:$VERSION` is available.
 - [ ] Publish the draft release prepared at the [Github releases page](https://github.com/kubernetes-sigs/kueue/releases).
-- [ ] Add a link to the tagged release in this issue.
+- [ ] Add a link to the tagged release in this issue:
```

Comment:
revert? or did you want to add something?

Reply:
added an example.